src/main/java/org/codehaus/plexus/archiver/zip/AbstractZipArchiver.java

```diff
@@ -530,6 +530,11 @@ public InputStream get()
 
     private void setTime( java.util.zip.ZipEntry zipEntry, long lastModified )
```

Comment:
Shouldn't a specific setter don't override a global value?

Reply:
it's up to the method writer to define the semantics
coding the global override in the helper just makes the change easy, and avoids missing any location
if you want, we can change the name of the variable (originalLastModified?): but I can't find a good one...

Reply:
Regardless of the name, this has to be clearly documented. Maybe an overloaded method with boolean could cover both cases?

Reply:
hey, it's a private helper that is quite short: don't do too much